pkgs/development/python-modules/rectangle-packer/default.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  cython_0,
+}:
+
+buildPythonPackage rec {
+  pname = "rectangle-packer";
+  version = "2.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
```

Comment:
The newest commit (rev = 46fa636fc8637081845151b4a0b16e8f60a57638) does not depend on Cython<3 anymore. There has unfortunately not been a release for it, yet, but it should unblock this PR.

Reply:
I just opened [an issue](https://github.com/Penlect/rectangle-packer/issues/30) on the repo to ask for a release

Reply:
Version 2.0.4 released!